gtsfm/averaging/rotation/rotation_averaging_base.py

```diff
@@ -14,14 +14,26 @@
 from gtsfm.common.pose_prior import PosePrior
 from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
 
+from gtsfm.ui.gtsfm_process import GTSFMProcess, UiMetadata
 
-class RotationAveragingBase(metaclass=abc.ABCMeta):
+
+class RotationAveragingBase(GTSFMProcess):
     """Base class for rotation averaging.
 
     This class generates global rotation estimates from the pairwise relative
     rotations.
     """
 
+    def get_ui_metadata() -> UiMetadata:
+        """Returns data needed to display this process in the process graph."""
+
+        return UiMetadata(
+            "Rotation Averaging",
```

Comment:
nit: could we use keyword args here for clarity?

Reply:
To clarify, you want something like this?
```
        return UiMetadata(
            display_name="Translation Averaging",
            parent_plate="Multi-View Optimizer",
            input_products=(
                "View-Graph Relative Translations",
                "Global Rotations",
                "Absolute Pose Priors",
                "Relative Pose Priors",
            ),
            output_products=("Global Translations"),
        )
```

Reply:
Yes -- thanks!